docs/tutorials/noise.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.      ret += cirq.H.on_each(*qubits)
</pre><p>is this really working? what about <code>ret += cirq.Circuit(cirq.H.on_each(*qubits)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/quantum/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>